torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -320,7 +282,8 @@ void ProfilingRecord::instrumentBlock(Block* block) {
 
 void ProfilingRecord::removeProfilingNodes(Block* b) {
   for (auto it = b->nodes().begin(); it != b->nodes().end(); it++) {
-    if (it->kind() == prim::profile || it->kind() == prim::profile_optional) {
+    if (it->kind() == prim::profile || it->kind() == prim::profile_optional ||
+        it->kind() == prim::profile_ivalue) {
```

Comment:
Could we remove all references to `profile_optional` ?

Reply:
Will do.